app.py

```diff
@@ -12,6 +12,8 @@
 app.config.from_pyfile('settings.py',silent=True)
 if "VAULT_ADDR" in os.environ:
     app.config['VAULT_URL'] = os.environ['VAULT_ADDR']
+if "VAULT_SKIP_VERIFY" in os.environ:
```

Comment:
VAULT_ADDR is setting the value to the environment variable.  You are hard setting the value to True.  Those 2 things are not equivalent.